pulp_rpm/tests/functional/content_handler/test_config_repo.py

```diff
@@ -0,0 +1,108 @@
+# coding=utf-8
```

Comment:
This is the Python3 default so it can be removed https://www.python.org/dev/peps/pep-3120/